lib/ui/widgets/calendar_item.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:kjg_muf_app/model/event.dart';
+
+import '../../model/calendar.dart';
+
+DateFormat dateFormat = DateFormat("dd.MM.yyyy");
+DateFormat timeFormat = DateFormat("HH:mm");
+
+Widget calendarItem(BuildContext context, Calendar calendar) {
+  return DataTable(
+    columns: List<DataColumn>.generate(
+        7,
```

Comment:
Für die Anzahl der Wochentage etc. könnte man Konstanten anlegen, falls sich der Anzahl in der Woche mal in der Zukunft ändert :P